src/main/java/mate/academy/until/ConnectionUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.until;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver.";
```

Comment:
The JDBC driver string has an extra period at the end. It should be 'com.mysql.cj.jdbc.Driver' without the trailing period.